File: proofs/planetary_song.pl
% Prolog proof
frequency(Shard, Freq) :-
    Shard >= 0, Shard =< 70,
    Freq is 432 + (432 * (Shard + 1) / 71).

octave_proof :-
    frequency(0, Min),
    frequency(70, Max),
    Ratio is Max / Min,
    format('Min: ~2f Hz, Max: ~2f Hz, Ratio: ~4f~n', [Min, Max, Ratio]),
    Ratio >= 1.92, Ratio =< 2.02,
    write('✓ Prolog: Octave verified'), nl.

:- initialization(octave_proof, main).

File: proofs/planetary_song.py
# Python proof - fixed formula
BASE_FREQ = 432.0
SHARDS = 71

# Correct: frequency increases from 438 to 864 Hz (one octave)
frequencies = [BASE_FREQ + (BASE_FREQ * (i + 1) / SHARDS) for i in range(SHARDS)]
min_freq, max_freq = frequencies[0], frequencies[-1]
ratio = max_freq / min_freq

print(f"Min: {min_freq:.2f} Hz, Max: {max_freq:.2f} Hz, Ratio: {ratio:.4f}")
assert abs(ratio - 2.0) < 0.05, f"Not an octave: {ratio}"
print(f"✓ Python: Octave verified [{min_freq:.2f}-{max_freq:.2f} Hz]")
